docsets/Love2d/docset.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "Love2d",
-    "version": "11.2",
+    "version": "11.3",
     "archive": "love2d.tgz",
```

Comment:
You need to update the archive at this path as well.

Reply:
Thanks. I just change it.